databricks/koalas/tests/test_dataframe.py

```diff
@@ -1079,27 +1082,79 @@ def test_cummin(self):
         self.assert_eq(pdf.cummin(), kdf.cummin())
         self.assert_eq(pdf.cummin(skipna=False), kdf.cummin(skipna=False))
 
+    @unittest.skipIf(
+        LooseVersion(pyspark_version) < LooseVersion('3.0'),
+        "columns axis is supported in Spark 3.0+.")
```

Comment:
Ah .. I forgot that struct as return type was only added lately as of https://github.com/apache/spark/commit/ddc2052ebd247aa2a8dad34fd5c1cd345fa45118#diff-c1cf83efe5a5b1f2f1f770589988e997 and https://issues.apache.org/jira/browse/SPARK-23836.

I tested all of them against Spark 3.0.